### PR TITLE
add netlify identity starter

### PR DESCRIPTION
**Pull request by lwz7512**
## Description

hi, I appended a new starter to the starters.yml, please check it out...

## Related Issues

no yet.

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1420,3 +1420,18 @@
     - Prettier integrated into ESLint
     - Jest for component testing
     - A few example components and pages with stories and simple site structure
+- url: https://inspiring-me-lwz7512.netlify.com/
+  repo: https://github.com/lwz7512/gatsby-netlify-identity-starter
+  description: Gatsby Netlify Identity Starter with NIW auth support, and content gating, as well as responsive layout.
+  tags:
+    - Netlify-identity-widget
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Netlify Identity
```

File: docs/starters.yml [hunk 2]
```diff
@@ -1420,3 +1420,18 @@
...
+    - User authentication
```
Comment (LekoArts) [inline docs/starters.yml]: Already covered by "Netlify Identity"

```suggestion
```

File: docs/starters.yml [hunk 3]
```diff
@@ -1420,3 +1420,18 @@
...
+    - Resposive layout
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 4]
```diff
@@ -1420,3 +1420,18 @@
...
+    - Pagination
+    - Private routing
```
Comment (LekoArts) [inline docs/starters.yml]: Covered by "Netlify Identity"

```suggestion
```

**Comment by lwz7512** [comment]
@LekoArts hi, I have made some changes including remove the yarn.lock,  is this OK? need anything to do before Merge ?

**Pull request merged**